mod/developers/views/default/theme_sandbox/intro.php

```diff
@@ -30,16 +30,24 @@
 	always recommended to disable caches while developing themes and plugins. To
 	disable caches, visit the $advanced or $developers pages.";
 
-	echo elgg_view_module('info', 'Warning', $body);
+	echo elgg_view_message('warning', $body);
 }
+
+$break_out_link = elgg_view('output/url', [
```

Comment:
Have you ever used theme sandbox in an iframe? Maybe just get rid of it

Reply:
we could just open the menu item in a blank target